library/modules/bigip_profile_server_ssl.py

```diff
@@ -186,6 +196,7 @@ class Parameters(AnsibleF5Parameters):
         'sniRequire': 'sni_require',
         'serverName': 'server_name',
         'peerCertMode': 'server_certificate',
+        'renegotiation': 'renegotiation',
```

Comment:
why is this here :) ?

Reply:
Consistency with other modules 😄

Reply:
this is not how you use api_map, please remove this :)